src/main/resources/SeleniumParallelTestSuite.xml

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="Parallel test suite" parallel="tests" thread-count="8" group-by-instances="true">
```

Comment:
If you have multiple test suites and if you'd like to run all suites (maybe that's the case) you can create suite of suites
like here
```<suite-files>
    <suite-file path="suites/DemoAppTestSuite.xml"/>
    <suite-file path="suites/SwaggerTestSuite.xml"/>
    <suite-file path="suites/PortalTestSuite.xml"/>
    <suite-file path="suites/DashboardsTestSuite.xml"/>
  </suite-files>```

Reply:
It doesn't apply here for now, but thanks for the tip for the future! :)